static/src/javascripts/projects/common/modules/identity/sign-in-gate/variants/control.js

```diff
@@ -24,11 +27,6 @@ const canShow: (name?: string) => boolean = (name = '') =>
     !isInvalidArticleType() &&
     !isInvalidSection();
 
-// method which runs if the canShow method returns true, used to display the gate and logic associated with it
-// it returns a boolean, since the sign in gate is based on a `Banner` type who's show method returns a Promise<boolean>
-// in our case it returns true if the method ran successfully, and false if there were any problems encountered
-const show: () => boolean = () => true;
```

Comment:
Is it worth importing the `show as designShow` and call `const show: () => boolean = designShow()` just so its more explicit what is going on ?

Reply:
that's a good idea! would help for the clarity